src/main/java/com/agilethought/atsceapi/service/UserServiceImpl.java

```diff
@@ -1,5 +1,51 @@
 package com.agilethought.atsceapi.service;
 
-public class UserServiceImpl {
+import com.agilethought.atsceapi.exceptions.UnauthorizedException;
+import com.agilethought.atsceapi.model.LoginData;
+import com.agilethought.atsceapi.model.User;
+import com.agilethought.atsceapi.repository.UserRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.List;
 
-}
+@Service
+@Slf4j
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Override
+    public User loginMethod(LoginData loginData) {
+
+        if (isValid(loginData.getEmail()) && isStringLowerCase(loginData.getEmail())) {

```

Comment:
There are some validations in this class that we should move them to a specific class only for validations, but for now we are ok with them, it will be a further implementation in other sprint.